src/mon/ConfigMap.cc

```diff
@@ -204,6 +204,20 @@ bool ConfigMap::parse_mask(
   return true;
 }
 
+void ConfigMap::print_value(Formatter *f, bufferlist &odata,
+                 const string &name, const string &value)
+{
+  if (f) {
+    f->open_object_section("config");
+    f->dump_string("name", name);
+    f->dump_string("value", value);
+    f->close_section();
+    f->flush(odata);
```

Comment:
Please also include the information from:

https://github.com/ceph/ceph/blob/d6a36b014bf1f447801ad009d0677022c61645c9/src/mon/ConfigMonitor.cc#L356-L362

and update that code to also call `print_value`.

Reply:
print_value includes mask,section,can_update_at_runtime, in mgr/DaemonServer.cc, it also includes  source, overrides. 
I don't know how to unify. Do you have any good ideas？